### DISCUSSION TITLE
MIssing ORT files

### DISCUSSION THREAD

**Discussion by eyalshaked** [DakeQQ/Native-LLM-for-Android#17] [General]
Thank you for this amazing project! I truly appreciate all the work that has gone into it. 

I was wondering, where can I find the ORT files for the Qwen2-VL 2B model? Specifically, are there pre-exported ONNX Runtime (ORT) files available for this model? Thank you!

**Comment by DakeQQ** [comment]
Sure, We will provide a download link later.

**Comment by DakeQQ** [comment]
Hello, the **QwenVL_2B** model is now [available](https://drive.google.com/file/d/1lmvg1rxQ1c33BWX-BnOwVY2R6xPhENvS/view?usp=drive_link). Please extract the files and place **QwenVL_A.ort**, **QwenVL_B.ort**, **QwenVL_C.ort**, **QwenVL_D.ort**, and **QwenVL_E.ort** into the **assets** folder.  

Additionally, update the variable `screen_resolution_short_side` in **GLRender.java** (line 37) to match your phone's screen width resolution. This ensures the camera view is properly centered on your phone screen.
